OlinkAnalyze/vignettes/plate_randomizer.Rmd

```diff
@@ -0,0 +1,116 @@
+
+
+```{r Outlier_data, include=FALSE}
+library(OlinkAnalyze)
+data(manifest)
+```
+
+
+## Sample randomization
+
+### Complete randomization
+
+The simplest way to randomize the samples is to perform complete randomization.
+
+```{r complete_randomization, message=FALSE}
+randomized.manifest_a <- olink_plate_randomizer(manifest, seed=12345)
+```
+
+
+### Generate randomization scheme that keeps subjects on the same plate
+
+In the case of multiple samples per subject (e.g. in longitudinal studies), Olink recommends keeping each subject on the same plate to further reduce variation in the data. The individuals should then be distributed so that the remaining experimental variables of interest are evenly distributed across plates. This could be achieved by using the `SubjectColumn` argument. The number of samples on each plate could be specified via the `available.spots` argument. For example, the following code will lead to 78 samples on the first plate, and 30 samples on the second and the third plates. The number of iterations for fitting subjects on the same plate can be set by the `iterations` argument.
+
+
+```{r subjects_randomization, message=FALSE}
+randomized.manifest_b <- olink_plate_randomizer(manifest,SubjectColumn="SubjectID", 
+                                                available.spots=c(78,30,30), 
+                                                iterations = 500,
+                                                seed=12345)
+```
+
+After randomization, the output would be a "tibble" including SampleID, SubjectID etc. assigned to well positions. Columns include same columns as Manifest with additional columns:
+
+  - plate: Plate number
+
+  - column: Column on the plate
+
+  - row: Row on the plate
+
+  - well: Well location on the plate
+
+## Visualization
+
+To illustrate the goodness of randomization, both `olink_displayPlateLayout()` and `olink_displayPlateDistributions` functions could be used.
+### Plate layouts
+
+The `olink_displayPlateLayout()` function could be used to visualize the layouts of the plate and specifying the color for the variable of interest using the `color_g` argument. The label of the colored variable could be shown in the plot via the `include.label` argument.
+
+```{r, fig.height = 6, fig.width = 6, fig.align = "center"}
+olink_displayPlateLayout(randomized.manifest_b, fill.color = 'SubjectID', include.label = FALSE)
+```
+
+The label of the colored variable could be shown in the plot via the `include.label` argument.
+
+```{r,fig.height = 6, fig.width = 6, fig.align = "center"}
```

Comment:
make it bigger than 6. right now the word "Empty" in the plot overlap between the cells, which dont look nice. If you change you might have to change above to keep consistency.